doc/integrator/admin_interface.rst

```diff
@@ -3,6 +3,10 @@
 Configure the admin interface
 =============================
 
+.. note::
+
+    This is not possible in the simple application mode
```

Comment:
Suggestion, be more explicit:
"Configuration of the admin interface is not possible in the simple application mode."